fili-core/src/test/groovy/com/yahoo/bard/webservice/data/HttpResponseMakerSpec.groovy

```diff
@@ -54,8 +56,13 @@ class HttpResponseMakerSpec extends Specification {
         paramMap = Mock(MultivaluedMap)
         DimensionDictionary dimensionDictionary = Mock(DimensionDictionary)
         Dimension dim1 = Mock(Dimension)
+        responseWriter = Mock(ResponseWriter)
 
-        httpResponseChannel = new HttpResponseChannel(Mock(AsyncResponse), apiRequest, new HttpResponseMaker(MAPPERS, dimensionDictionary))
+        httpResponseChannel = new HttpResponseChannel(Mock(AsyncResponse), apiRequest, new HttpResponseMaker(
```

Comment:
When anything needs to be wrapped, everything should be wrapped. So instead of having the first two arguments on the first line, and then wrapping the last, every argument should be on its own line:

```java
     new HttpResponseChannel(
                Mock(AsyncResponse),
                apiRequest,
                new HttpResponseMaker(...)
      )
```